spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/LSTM.scala

```diff
@@ -72,14 +90,20 @@ class LSTM[T : ClassTag] (
           .add(Dropout(p))
           .add(Dropout(p)))
         .add(ParallelTable()
-          .add(Linear(hiddenSize, hiddenSize, withBias = false))
-          .add(Linear(hiddenSize, hiddenSize, withBias = false))
-          .add(Linear(hiddenSize, hiddenSize, withBias = false))
-          .add(Linear(hiddenSize, hiddenSize, withBias = false)))
+          .add(Linear(hiddenSize, hiddenSize,
+            withBias = false, wRegularizer = uRegularizer))
+          .add(Linear(hiddenSize, hiddenSize,
+            withBias = false, wRegularizer = uRegularizer))
+          .add(Linear(hiddenSize, hiddenSize,
+            withBias = false, wRegularizer = uRegularizer))
+          .add(Linear(hiddenSize, hiddenSize,
+            withBias = false, wRegularizer = uRegularizer)))
         .add(JoinTable(1, 1))
     } else {
-      i2g = Linear(inputSize, 4 * hiddenSize)
-      h2g = Linear(hiddenSize, 4 * hiddenSize, withBias = false)
+      i2g = Linear(inputSize, 4 * hiddenSize,
```

Comment:
Fixed regularization Configuration for LSTM, wRegularizer for input weight, uRegularizer for hidden weight, bRegularizer for bias.